### ISSUE TITLE
Better way of getting player input into variables

### ISSUE THREAD

**Issue by josephcsible** [CyclopsMC/IntegratedDynamics#213]
Currently, there's no way of using player input in the variable system other than by putting the input in a logic programmer and putting the resulting card in a proxy. I see two better ways that this could be accomplished:
1. On the proxy, have the same interface as the logic progammer, so that values can be proxied directly instead of needing to make a variable card with them first
2. Add inputs to go along with screens as outputs. For example, a keyboard could have a string aspect.

**Comment by rubensworks** [comment]
I'm not sure what you are trying to accomplish exactly? A system that takes text input from the player and shows it on a display panel for instance?

**Comment by josephcsible** [comment]
That would be one possible use-case, yes. What I had in mind is that a player could type in a string like "diamond", and then all items with "diamond" in their name from an inventory would get sent to an output chest.

**Comment by rubensworks** [comment]
Yes, I get what you mean. I'll have to think about this a bit, to figure out a good way of designing something like this.
